prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -22,184 +23,189 @@ elements:
     csvFields: [{name: Patient_ID}]
```

Comment:
This will, I hope, create a merge conflict.   I changed this back to standard.patient_id (or, now, simply patient_id), to fix the bug that Patient_ID was missing in AZ and Pima data.

Suggest we leave it that way until we have a conversation with SimpleReport team on IDs.